pkgs/servers/monitoring/prometheus/influxdb-exporter.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub, nixosTests }:
+
+buildGoModule rec {
+  pname = "influxdb_exporter";
+  version = "0.8.0";
+  rev = "v${version}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "prometheus";
+    repo = "influxdb_exporter";
+    sha256 = "sha256-aNj4ru3yDet+jdcEpckFVaymmjWmKzTMPcTxPMNFbgo=";
+  };
+
+  vendorSha256 = null;
+
+  buildFlagsArray = let
+    goPackagePath = "github.com/prometheus/influxdb_exporter";
```

Comment:
This variable is nowhere used. Also ldflags should be used instead of buildFlagsArray.

Reply:
Copy/paste from node-exporter. :man_shrugging: